### ISSUE TITLE
Stats are reported wrong on ft.info

### ISSUE THREAD

**Issue by MeirShpilraien** [RediSearch/RediSearch#1497]
```
inverted_sz_mb
4.653799373510017e+18
offset_vectors_sz_mb
4.6393374363699118e+18
doc_table_size_mb
4.6545662448867738e+18
key_table_size_mb
4.6501903216179937e+18
```
Reproduce during benchmark check on 1.99.5

**Comment by ashtul** [comment]
ref #591 
ref #1186

**Comment by ashtul** [comment]
fixed by #1517 and #1522
